### ISSUE TITLE
サーベイ

### ISSUE THREAD

**Issue by Daikii127** [Daikii127/WM_group1#3]
### 📄 資料
+ 

### ✅ 作業
+ [ ] 

### 🚀 想定作業時間・期限
+

**Comment by Daikii127** [comment]
世界モデルを用いたモデルベース強化学習における 

DayDreamerV2 

- 状態表現を離散表現にしている
- ELBOのKLダイバージェンスのバランスをうまく学習できるようにした

DayDreamerV3

- 入力の再構成や報酬や価値の予測のスケールをドメインで揃えるためにsymlog予測を用いた
- KLのバランスをうまく学習できるように調整

HarmonyDream

- 観測や報酬の損失のバランスを調整することで学習効率を上げる
→観測と報酬の各タスクが優勢になりすぎず、モデルベース強化学習の効率を最大化することができる

TransDremaer
- RNNの代わりにTransformerを用いた状態空間モデル
→RNNは逐次的に時系列を学習する必要がある、Transformerを用いることで学習時に並列に計算することができる、性能も良い

TWM
- Transformerへの入力を離散オートエンコーダで獲得した潜在状態、行動、報酬を用いてTransformerで構成された遷移モデルに入力し決定論的な隠れ状態を予測

IRIS
- VQVAEで獲得した離散的な潜在空間上でダイナミクスをTransformerによって学習する

STORM
- 離散オートエンコーダで獲得した潜在状態と行動をMLPで結合しtransformerに入力し隠れ潜在状態を予測

Δ-Iris

- 観測を再構成する際に、潜在状態表現だけでなく行動と観測も加えることでzは差分を表すΔ-tokenとなる

REM

- TransformerとRNNを組みわせ良いとこどりをした、計算時間、性能の向上

masked world model


課題

- 性能

   - 観測や報酬のLOSSのバランスを取るのが難しい( Dreaner系の世界モデル)

- 計算時間

   - TSSMに関しては計算量が系列

Transformerと相性の良い入力

- 言語モデルで使われていたという背景から離散のトークンに強い

**Comment by Daikii127** [comment]
OpenVLA
https://arxiv.org/pdf/2406.09246

概要

- 70億のパラメータを持つvision-language-action model(VLA)
- Open X-Embodiment datasetから970,000件のエピソードを学習

**Comment by Daikii127** [comment]
VIMA
https://arxiv.org/pdf/2210.03094

abstract
- 幅広いロボット操作タスクをテキストと視覚トークンを組み合わせたマルチモーダルプロンプトで表現
- 新しいシミュレーションベンチマーク
- VIMAは競合する設計よりも最大2.9倍のタスク成功率を達成
- トレーニングデータを10分の1に削減しても、最良の競合モデルよりも2.7倍優れた性能

**Comment by Daikii127** [comment]
https://eureka-research.github.io/

**Comment by Daikii127** [comment]
Mastering Robot Manipulation with Multimodal Prompts through Pretraining
and Multi-task Fine-tuning
https://arxiv.org/pdf/2310.09676
